paparazzi/libs/layoutlib/build.gradle

```
@@ -17,22 +18,26 @@ tasks.register('cloneLayoutlib') {
     if (repoDir.exists() && !repoDir.list()) {
       repoDir.delete()
     }
+    Grgit grgit
     if (!repoDir.exists()) {
       logger.warn('Cloning prebuilt layoutlib: this make take a few minutes...')
```

```suggestion
      logger.warn('Cloning prebuilt layoutlib: this may take a few minutes...')
```

unrelated, but apparently future me catches past me typos 😄